pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -336,7 +336,7 @@ func manageSecrets(nsList []string, h *helper.H) error {
 		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("failed to create secret %s in namespace %s", newSecretName, ns))
 
 		// check 'get' permission
-		dummySecret, err = secrets.Get(context.TODO(), newSecretName, metav1.GetOptions{})
+		dummySecret, err := secrets.Get(context.TODO(), newSecretName, metav1.GetOptions{})
```

Comment:
as in line 303

```suggestion
		_, err := secrets.Get(context.TODO(), newSecretName, metav1.GetOptions{})
```

Reply:
So basically, this is not related to my change at all.
Just the `make` failed locally due to the golang lint failed.
The reason for line 330 is that the `dummySecret` declared and assigned but not being used at all. Or the value has been overridden by this line.

But the value of `dummySecret` is required since it is being used in several lines below. Line 347